---
title: LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER | Документация Майкрософт
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: c4288265-43f7-44fd-acab-ba64d04264d9
author: stevestein
ms.author: sstein
manager: craigg
ms.openlocfilehash: be37ed399f5beed087c71b9aaf31154615116d24
ms.sourcegitcommit: 61381ef939415fe019285def9450d7583df1fed0
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 10/01/2018
ms.locfileid: "47792782"
---
# <a name="localdberrorcannotaccessinstancefolder"></a>LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
    
## <a name="details"></a>Сведения  
  
|||  
|-|-|  
|Название продукта|SQL Server|  
|Идентификатор события|261|  
|Источник события|Среда выполнения локальной базы данных SQL Server 12.0|  
|Компонент|API среды выполнения локальной базы данных|  
|Текст сообщения|Не удается получить доступ к папке экземпляра локальной базы данных: %%LOCALAPPDATA%%\Microsoft\Microsoft SQL Server локальный DB\Instances\\< имя экземпляра\>.|  
  
## <a name="explanation"></a>Объяснение  
 Не удалось получить доступ к папке экземпляра.  
  
## <a name="user-action"></a>Действие пользователя  
  
